shared/build.gradle.kts

```diff
@@ -35,6 +35,11 @@ kotlin {
   }
 }
 
+configurations.configureEach {
+  // Remove unnecessary dependency of Precompose and Moko
```

Comment:
Should we file a bug there?

Reply:
Yes, as far as I can tell these libraries depend on Core, Activity, or Lifecycle but not on AppCompat directly.

It's an old habit to just add AppCompat as a dependency with all the dependencies it gets along but if the code doesn't use Views and doesn't directly depend on any code in the package androidx.appcompat, it's a mistake.

Reply:
* https://github.com/Tlaster/PreCompose/pull/316
* https://github.com/icerockdev/moko-resources/pull/687